### DISCUSSION TITLE
Flow Variables

### DISCUSSION THREAD

**Discussion by alexLaws** [postmanlabs/postman-flows#254] [Defect]
I wrote a flow within the last 2 weeks that was able to capture a variable from within the JSON response to a request, and then pass that variable to another query. In the same flow, I also created a durable by pulling a variable from a different JSON response.

I went to review my work this morning and it seems to me that the flow system is no longer parsing the JSON. I used to be able to dive into the body of the response (e.g path: /data/body/data/key) but now I can't click past "body" and the old request information details display the variables as errors because the json doesn't appear to be populating anymore (see image). ![image](https://user-images.githubusercontent.com/8510416/154166867-08c01ef8-f96f-410f-a211-80e6c86c5ddb.png)

I believe I am following the same steps so I think the system must have changed in some way, but I can't be 100% sure. Any guidance would be helpful.

**Comment by saswatds** [comment]
Hey @alexLaws Do you have any examples defined for your Request? Flow's is probably looking for an Example to understand how your data looks like.

**Reply by alexLaws** [reply]
No I do not. However, I opened postman yesterday to use it and I am pretty sure I saw a pop up the first time I loaded the flow asking me to set a data example. I didn't realize what I was doing and clicked away from it and then couldn't find it again. 

Could you please share how to set an example? Thank you for the reply!

**Reply by saswatds** [reply]
https://github.com/postmanlabs/postman-flows/discussions/236#discussioncomment-2087323 I think this comment would be helpful.  You can find a much detailed explanation here https://learning.postman.com/docs/sending-requests/examples/

**Reply by alexLaws** [reply]
This works. Thank you!!
